osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/services/MonitoringService.java

```diff
@@ -112,10 +113,44 @@ public void handleGetPQValuesResponse(final GetPQValuesResponseDto getPQValuesRe
             actualCorrelationUid = getCorrelationId("DeviceGenerated", deviceIdentification);
         }
 
-        ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
+        final ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
                 .withCorrelationUid(correlationUid).withOrganisationIdentification(organisationIdentification)
                 .withDeviceIdentification(deviceIdentification).withResult(result).withOsgpException(osgpException)
                 .withDataObject(getPQValuesResponse).build();
         this.webServiceResponseMessageSender.send(responseMessage, messageType);
     }
+
+    public void handleGetMeasurementReportResponse(final MeasurementReportDto measurementReportDto,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 7 parameters, which is greater than 6 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS00107)